x-pack/plugins/ml/public/application/data_frame_analytics/common/analytics.ts

```diff
@@ -65,6 +65,12 @@ export interface LoadExploreDataArg {
   searchQuery: SavedSearchQuery;
 }
 
+export interface MetricItem {
```

Comment:
Should this be named e.g. `ClassificationMetricItem` or even `ClassificationEvaluationMetricItem` as outlier and regression jobs have different evaluation metrics?